src/components/Profile/InactiveProposalsStake.tsx

```diff
@@ -24,44 +31,52 @@ function InactiveProposalsStake({ myInactiveStakes }) {
   )
   return (
     <Box heading="Inactive proposals stake" padding={3 * GU}>
-      {myInactiveStakes.map((stake) => {
+      {myInactiveStakes.map((stake, index) => {
         return (
-          <ProposalItem
-            amount={stake.amount}
-            compact={compact}
-            gardenId={stake.proposal.organization.id}
-            proposalId={stake.proposal.id}
-            proposalName={stake.proposal.metadata}
-            selectProposal={handleSelectProposal}
-          />
+          <div
+            key={index}
+            css={`
+              display: flex;
+              align-items: center;
+              margin-bottom: ${1 * GU}px;
+            `}
+          >
+            <ProposalItem
+              amount={stake.amount}
+              gardenId={stake.proposal.organization.id}
+              proposalId={stake.proposal.id}
+              proposalName={stake.proposal.metadata}
+              compact={compact}
+              selectProposal={handleSelectProposal}
+            />
```

Comment:
What do you think of also grouping the inactive proposals per Garden?

Reply:
The data in the inactive stake is slightly different, but I can do it. 👌

Reply:
One question @0xGabi . should the inactive one be included on the calculation of the percentage?

Reply:
I think they should as they are part of the token balance the use have available for voting.

Reply:
I will add the inactive. But they need to have something to distinguish from the active one. What do you suggest? I could add a separator.